### PR TITLE
enhance/copy-watchlist-popup

### PR DESCRIPTION
**Pull request by haritonasty**
**Summary**
1) Added existing user watchlists
2) rewrote to new styles

Guys, sorry for big PR)

**Screenshots**
also I've added styles for mobile, but not implemented on mobile watchlist page yet. 
In next PR, with adding whole watchlist context menu
in action

### CODE REVIEW

File: src/components/Search/SearchProjects.js [hunk 1]
```diff
@@ -1,19 +1,24 @@
 import React from 'react'
 import { graphql } from 'react-apollo'
 import { Icon, SearchWithSuggestions } from '@santiment-network/ui'
+import { Checkbox } from '@santiment-network/ui/Checkboxes'
```
Comment (haritonasty) [inline src/components/Search/SearchProjects.js]: we don't have default import for `Checkbox` in `sun-ui`

File: src/components/WatchlistCopy/WatchlistCopyPopup.js [hunk 2]
```diff
@@ -1,37 +1,215 @@
...
+
+  const open = () => setIsShown(true)
+
+  const normalizeListItems = items => items.map(({ project: { id } }) => id)
+
+  const checkRemainingAssets = (listId, assets) => {
+    const list = lists.find(({ id }) => listId === id)
+    const listItems = list ? list.listItems : []
+
+    const remainingAssets = [...assets].filter(id => {
```
Comment (DmitriVanGuard) [inline src/components/WatchlistCopy/WatchlistCopyPopup.js]: What is the reason for creating new array? `filter` returns the new array without mutating the old one

File: src/components/WatchlistCopy/WatchlistCopyPopup.module.scss [hunk 3]
```diff
@@ -1,42 +1,116 @@
...
+
+  @include responsive('phone-xs', 'phone') {
+    margin-right: 20px;
+  }
+}
+
+.heading {
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: bold;
```
Comment (DmitriVanGuard) [inline src/components/WatchlistCopy/WatchlistCopyPopup.module.scss]: `@include text('caption', 'm')`

File: src/components/WatchlistCopy/WatchlistCopyPopup.module.scss [hunk 4]
```diff
@@ -1,42 +1,116 @@
...
   text-align: center;
-  margin: 0;
-  padding: 10px;
-  border-bottom: 1px solid #eee;
+  padding: 5px 16px;
+
+  @include text('caption');
+
+  @include responsive('laptop', 'tablet', 'desktop') {
+    padding: 16px 16px 0 16px;
```
Comment (DmitriVanGuard) [inline src/components/WatchlistCopy/WatchlistCopyPopup.module.scss]: Could be written as a shorthand `16px 16px 0`

File: src/components/WatchlistCopy/WatchlistCopyPopup.js (hunk 2 above)
Comment (haritonasty) [reply src/components/WatchlistCopy/WatchlistCopyPopup.js]: `assets` has `Set` type

File: src/components/Search/SearchProjects.js [hunk 5]
```diff
@@ -32,12 +35,20 @@ const SearchProjects = ({
...
-                ticker={ticker}
-                name={name}
-              />{' '}
+              {isCopyingAssets && (
+                <Checkbox
+                  isActive={checkedAssets.has(id)}
+                  className={styles.checkbox}
+                />
+              )}
+              {!isCopyingAssets && (
```
[the remark this answers is not in the document]
Comment (Partysun) [reply src/components/Search/SearchProjects.js]: +

**Pull request merged**